content/balance/Compendium.md

```diff
@@ -405,31 +410,29 @@ Damage wise, with 100% uptime on all of the Phials and not counting in the downs
 
 </div>
 
-[Hissing Rune](https://www.wowhead.com/item=204973/hissing-rune): Mastery rune<br>
-Use this for every situation.
+Our main rune is [Hissing Rune](https://www.wowhead.com/item=204973/hissing-rune): Mastery rune<br>
+In some rare cases [Sophic Rune](https://www.wowhead.com/item=194820/howling-rune) might sim slightly better.
```

Comment:
whats sophic rune and are they really rare cases
should just mention them buzzing as well since BSL devalues more mastery a bit